### ISSUE TITLE
Question about training

### ISSUE THREAD

**Issue by JacksonCakes** [omriav/blended-diffusion#5]
Hi, this is really an impressive work! Two question here.
1. I would like ask is the overall process of the text-guided image editing is using only pre-trained model without any extra training or fine-tuning?
2. If it does not required any further fine-tuning or training, what is the purpose of having diffusion guided loss (which combine loss from CLIP model and background preservation loss)?

Thanks in advance for your clarification!

**Comment by JacksonCakes** [comment]
Thanks for you quick respond!
I think I should better rephrase my question 2. 
I can see that you are using CLIP loss at each reverse denoising step to better guide the generation of seamless output between background and edited region. So when does the diffusion guided loss which is the combination of CLIP loss and background preservation loss in Algorithm 1 come into play?

**Comment by omriav** [comment]
Algorithm 1 is a weak baseline that we added in the paper and showed that algorithm 2 (AKA Blended Diffusion) produce better results with no need for background preservation loss.

**Comment by JacksonCakes** [comment]
Alright, thanks for your clarification!
